pages.de/osx/sed.md

```diff
@@ -0,0 +1,36 @@
+# sed
+
+> Führe grundlegende Texttransformationen an der Eingabe aus.
+> Weitere Informationen: <https://ss64.com/osx/sed.html>.
+
+- Ersetzt das erste Vorkommen einer Zeichenkette in einer Datei und gibt das Ergebnis aus:
+
+`sed 's/{{finde}}/{{ersetze}}/' {{dateiname}}`
```

Comment:
```suggestion
- Ersetzt das erste Vorkommen einer Zeichenkette in einer Datei und gibt das Ergebnis aus:

`sed 's/{{finde}}/{{ersetze}}/' {{Pfad/zu/Datei}}`
```

Required fix: [broken placeholder](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths): `path/to/*` translated generic placeholder expected.